### PR TITLE
fix(Popover): fix gap under the triangle

### PR DESCRIPTION
**Pull request by TengFeiHao**
### Before submitting a pull request, please make sure the following is done:

1. Read the [contributing guide](https://github.com/youzan/vant/blob/dev/.github/CONTRIBUTING.md).
2. If you've added code that should be tested, add tests.
3. If you've changed APIs, update the documentation.
4. Ensure the test suite passes (`npm test`).

#### Title Format

type(ComponentName?)：commit message

Example：

- docs: fix typo in quickstart
- build: optimize build speed
- fix(Button): incorrect style
- feat(Button): add color prop

Allowed Types:

- fix
- feat
- docs
- perf
- test
- types
- build
- chore
- refactor
- breaking change

### CODE REVIEW

**Comment by chenjiahan** [comment]
Can you provide the steps to reproduce the problem?

**Comment by TengFeiHao** [comment]
https://github.com/TengFeiHao/vant-popover
image

**Comment by chenjiahan** [comment]
了解了👌

File: packages/vant/src/popover/index.less [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 :root {
   --van-popover-arrow-size: @popover-arrow-size;
+  --van-popover-arrow-offset-size: @popover-arrow-offset-size;
```
Comment (chenjiahan) [inline packages/vant/src/popover/index.less:5]: 这里不建议新增一个变量，是不是直接使用 `--van-popover-arrow-size` 也可以

Comment (TengFeiHao) [reply packages/vant/src/popover/index.less]: 可以，已经改好提交了。

**Review by chenjiahan — APPROVED** [review]

**Comment by chenjiahan** [comment]
👍

**Pull request merged**